<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab06_TNR</class>
 <widget class="QMainWindow" name="Lab06_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab05_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your polynom:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="aInput1_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bInput1_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cInput1_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="setPolynom1Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set polynom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>c:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="derivativeBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Derivative</string>
    </property>
   </widget>
   <widget class="QPushButton" name="integralBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Integral</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your derivative:</string>
    </property>
   </widget>
   <widget class="QLabel" name="derivativeRes_tnr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="derivativeX_tnr">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Roots</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Derivative</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your integral:</string>
    </property>
   </widget>
   <widget class="QLabel" name="integralRes_tnr">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Definite Integral</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval start:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="intervalStart_tnr">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="intervalEnd_tnr">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval end:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Second polynom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bInput2_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cInput2_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>c:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="aInput2_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>b:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setPolynom2Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set polynom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Difference:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add second polynom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtractBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtract second polynom</string>
    </property>
   </widget>
   <widget class="QLabel" name="sumRes_tnr">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2x^2</string>
    </property>
   </widget>
   <widget class="QLabel" name="difRes_tnr">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="polynom1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QLabel" name="polynom2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="calculateX_tnr">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="calcRes_tnr">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inverseBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Inverse first polynom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multBy">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>430</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>multiply by</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="coeff_tnr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="areEqual_tnr">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Press &quot;Are they equal?&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="areEqualBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Are they equal?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polynomsCountBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>How many polynoms were created?</string>
    </property>
   </widget>
   <widget class="QLabel" name="polynomsCount_tnr">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
